lib/thor/actions/empty_directory.rb

```diff
@@ -112,11 +112,18 @@ def invoke_with_conflict_check(&block)
         if exists?
           on_conflict_behavior(&block)
         else
-          say_status :create, :green
```

Comment:
If `say_status` is before `yield`, in case of a conflict we have a misleading message. For instance:
```
create  doc/config/config.rb
file_clash  doc/config/config.rb
```